### PR TITLE
[JS_HW][JS_2]

### PR DESCRIPTION
**Pull request by JackBlaaack**
homework with topic "if"

### CODE REVIEW

File: JS_2/task1.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+const age_2 = 18;
+
+// Создать переменную “age_3” и присвоить ей значение 60
+
+const age_3 = 60;
+
+// Создать if в котором будите проверять значение переменной age
+
+if(age === 10) {
```
Comment (anatoly-karpovich) [inline JS_2/task1.js:15]: don't see any requirements for this row. Is it needes?

File: JS_2/task3.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+// Преобразовать Task 2* таким образом, чтобы age принимался через prompt в привязанной вёрстке, а результат выводился в alert
+const age = +prompt("Enter your age, please")
```
Comment (anatoly-karpovich) [inline JS_2/task3.js:2]: No checks for invalid value from prompt. Please provide validations according to requirements

**Review by anatoly-karpovich — CHANGES_REQUESTED** [review]

File: JS_2/task3.js (hunk 2 above)
Comment (JackBlaaack) [reply JS_2/task3.js:2]: do you mean type of value, more than zero?

File: JS_2/task1.js (hunk 1 above)
Comment (JackBlaaack) [reply JS_2/task1.js:15]: maybe check type, and another condition else

File: JS_2/task2.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    console.log("Welcome!");
+    break;
+
+  case (age > age_3):
+    console.log("Keep calm and look Culture channel");
+    break;
+
+  default:
+    console.log("Technical work");
+}
```
Comment (VladimirLQA) [inline JS_2/task2.js:33]:
```suggestion
3. Преобразовать Task 2 - 1 таким образом, чтобы значение НАПРИМЕР '2' (т.е. ЛЮБАЯ строка в которой лежат ТОЛЬКО ЦИФРЫ) пропускалось,
    преобразовываясь в number
```

Also I'd like to see the solution to this subtask

File: JS_2/task3.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
+// Преобразовать Task 2* таким образом, чтобы age принимался через prompt в привязанной вёрстке, а результат выводился в alert
+const age = +prompt("Enter your age, please");
+const age_2 = 18;
+const age_3 = 60;
+switch (true) {
+  case 0:
+  case -1:
+  case "my age":
+    console.log("Invalid age");
```
Comment (VladimirLQA) [inline JS_2/task3.js:9]: Probably you need add some comparisions to get in this case

Why we check exactly this cases `0, -1, 'my age'` ? I will input `-2` or left it empty, for example
`isNaN` can help you achieve the correct result in `if...else` statement before comparing the ages in `switch`

**Review by VladimirLQA — CHANGES_REQUESTED** [review]

File: JS_2/task3.js (hunk 4 above)
Comment (JackBlaaack) [reply JS_2/task3.js]: thanks for advice)

File: JS_2/task2.js (hunk 3 above)
Comment (JackBlaaack) [reply JS_2/task2.js:24]: I got it

**Review by VladimirLQA — APPROVED** [review]

**Review by anatoly-karpovich — APPROVED** [review]

**Pull request merged**